docs/content/dotnet-core.md

```diff
@@ -161,6 +162,13 @@ the policy please refer to this
 [documentation](https://docs.microsoft.com/en-us/dotnet/core/versions/selection)
```

Comment:
What's the purpose of this section?

Reply:
Pointing the user to the roll forward logic for version selection that we use in our buildpack